internal/cmd/cli/apply/apply.go

```diff
@@ -218,20 +218,20 @@ func Dir(ctx context.Context, client Getter, name, baseDir string, opts *Options
 	bundleID := filepath.Join(name, baseDir)
 	bundleID = names.HelmReleaseName(bundleID)
 
-	bundle, scans, err := readBundle(ctx, bundleID, baseDir, opts)
+	bundle, scans, err := newBundle(ctx, bundleID, baseDir, opts)
 	if err != nil {
 		return err
 	}
 
-	def := bundle.DeepCopy()
-	def.Namespace = client.GetNamespace()
+	bundle = bundle.DeepCopy()
```

Comment:
Why do a deep copy if we're not keeping the original?

Reply:
Right, I wanted to test if it can be removed. Since this is wrangler's k8s client, I think the returned resources are pointers to the cache. Modifying their content might modify the cached content,  too.